test/gas-proxy/execute_transaction_test.go

```diff
@@ -0,0 +1,18 @@
+package gas_proxy_test
```

Comment:
Should this be completed somehow?

Reply:
Yes, it's empty because the gas token doesn't work in ethertest backend. I'm trying to figure out why and then I'll complete the tests.